images/helm-operator/tests/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "helm" {
+  name  = "helm-operator-${random_pet.suffix.id}"
+  chart = "https://bsgrigorov.github.io/helm-operator/helm-operator-0.0.2.tgz"
+
+  values = [
+    jsonencode({
+      image = {
+        operator   = "data.oci_string.ref.registry_repo"
```

Comment:
according to the helm chart you provide, there is no `tag:` in the values file so I think it has to be something like the following:

> operator = data.oci_string.ref.registry_repo:data.oci_string.ref.pseudo_tag

Reply:
https://artifacthub.io/packages/helm/helm-operator/helm-operator?modal=values